pkg/retriever/combinators/asyncsplitter.go

```diff
@@ -0,0 +1,89 @@
+package combinators
```

Comment:
this file has the github syntax highlighter very confused!

Reply:
they really haven't updated it very well